### ISSUE TITLE
Back to top button doesn't have content

### ISSUE THREAD

**Issue by benoitlord** [benoitlord/portfolio_nuxt#7]
### Issue Description
Vérifier que les liens ont un texte perceptible

### Element location
`#bouton-haut`

### Element source
`<a data-v-1c0ebc08="" id="bouton-haut" href="#" role="complementary" class="visible">`

### To solve this violation, you need to...
Fix at least one (1) of these issues:
- L’élément n’a aucun contenu textuel visible par les lecteurs d’écran
- L’attribut aria-label n’existe pas ou est vide
- L’attribut aria-labelledby n’existe pas, fait référence à des éléments qui n’existent pas ou à des éléments vides ou non visibles
- L’élément n’a pas d’attribut title ou l’attribut title est vide.

And

Fix the following:
- L’élément est dans l’ordre de tabulation et n’a pas d’intitulé accessible
